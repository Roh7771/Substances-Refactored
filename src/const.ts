export const SUBSTANCE_TO_SHOW_AMOUNT = 20;

export const EMPTY_FIELD_VALUE = '-';

export enum ErrorStatus {
  OK,
  DUPLICATE_CAS_NUMBER,
  LOADING_FAILED,
  WRONG_LOGIN_DATA
}

export enum AuthorizationStatus {
  AUTH,
  NO_AUTH
}

export enum ModalWindowStatus {
  EDIT = 'EDIT',
  CREATE = 'CREATE',
  DELETE = 'DELETE',
  NONE = 'NONE'
}
