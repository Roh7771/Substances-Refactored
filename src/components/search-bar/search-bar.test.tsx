import * as React from 'react';
import * as renderer from 'react-test-renderer';
import SearchBar from './search-bar';
import { ErrorStatus } from '../../const';

const noop = () => null;

it('Search bar should render correctly', () => {
  const tree = renderer.create(
    <SearchBar
      searchSelectValue="some value"
      searchInputValue="some value"
      errorStatus={ErrorStatus.OK}
      dispatch={noop}
    />,
  );
  expect(tree).toMatchSnapshot();
});
