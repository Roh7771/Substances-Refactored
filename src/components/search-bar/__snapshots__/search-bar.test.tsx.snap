// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search bar should render correctly 1`] = `
<nav
  className="navbar navbar-light bg-light"
>
  <span
    className="navbar-brand font-weight-bold"
  >
    Лаборатория СМАРТ-НАНОСИСТЕМ
  </span>
  <form
    className="form-inline"
    onSubmit={[Function]}
  >
    <select
      className="form-control"
      disabled={false}
      onChange={[Function]}
      required={true}
      style={
        Object {
          "marginRight": "20px",
        }
      }
      value="Название"
    >
      <option>
        CAS RN
      </option>
      <option>
        Название
      </option>
    </select>
    <input
      aria-label="Search"
      className="form-control mr-sm-2"
      disabled={false}
      onChange={[Function]}
      placeholder="Поиск"
      style={
        Object {
          "width": "400px",
        }
      }
      type="search"
      value="some value"
    />
  </form>
</nav>
`;
